core/src/main/scala/kafka/admin/BrokerApiVersionsCommand.scala

```diff
@@ -72,10 +70,11 @@ object BrokerApiVersionsCommand {
                                    .withRequiredArg
                                    .describedAs("server(s) to use for bootstrapping")
                                    .ofType(classOf[String])
-    val options = parser.parse(args : _*)
+    options = parser.parse(args : _*)
     checkArgs()
 
     def checkArgs() {
+      CommandLineUtils.printHelpAndExitIfNeeded(this, "Command to retrieving broker version information.")
```

Comment:
Should we use a similar description format for all these tools? AclCommand.scala uses `This tools helps ...` while this one uses `Command to retrieving ...`. I personally prefer the former but can you pick one and apply it everywhere?